src/endpoints/accounts/account.controller.ts

```diff
@@ -138,11 +141,15 @@ export class AccountController {
   @ApiQuery({ name: 'isSmartContract', description: 'Return total smart contracts count', required: false })
   @ApiQuery({ name: 'name', description: 'Filter accounts by assets name', required: false })
   @ApiQuery({ name: 'tags', description: 'Filter accounts by assets tags', required: false })
+  @ApiQuery({ name: 'excludeTags', description: 'Exclude specific tags from result', required: false })
+  @ApiQuery({ name: 'hasAssets', description: 'Return a list accounts with assets', required: false })
```

Comment:
Return a list accounts with assets => Returns a list of accounts that have assets

Reply:
fixed